edd_utils/__init__.py

```diff
@@ -13,16 +13,25 @@
 from tqdm.auto import tqdm
 
 
-def login(edd_server='edd.jbei.org', user=None):
-    '''Log in to the Electronic Data Depot (EDD).'''
-    
+def login(edd_server='edd.jbei.org', user=None, default_to_system_user=True):
```

Comment:
I wonder if 'True' is a good default for `default_to_system_user` as it would not be useful when run through a docker for example. What do you think?

Reply:
I think it's hard to predict how people will use this API, since we can't necessarily know t how many end users will be using Docker.  Since we've already published this API with default behavior equivalent to setting `default_to_system_user=True`, I think we should leave that behavior unchanged to avoid breaking clients' code.